### ISSUE TITLE
DatePicker doesn't work with React 16.4

### ISSUE THREAD

**Issue by spyx08** [Hacker0x01/react-datepicker#1411]
### Expected behavior
I already used Datepicker in a lot of different app and got no problems
### Actual behavior
Now with React 16.4, when I try to render a normal DatePicker, app's crash. 

```
render(){
<DatePicker selected={moment()} onChange={this.onChange} />
}
```

I got this error 

> Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object

I tried with version 1.4.1 (who was working in a older app) and with 1.5.0 and no one is working.

**Comment by seansfkelley** [comment]
This type of message usually indicates that you've imported the library improperly and ended up with something like `{ default: DatePicker }` rather than just `DatePicker`. Maybe your tooling or build changed at the same time you upgraded React?

**Comment by spyx08** [comment]
Yes i was thinking it was bad imported/builded but I tried to remove all my node_modules and to re-install them with yarn and npm but got the same issue. 

But yes I expect an error with the import/build I don't know how I can check this...

**Comment by matttk** [comment]
More than half a year old. Anybody fix this problem? I have the same one. Yes, I am doing `import Datepicker`.

**Comment by matttk** [comment]
Just an update for anybody who stumbles upon this problem. I noticed the error was with `popper` and went over to their GitHub, where some people had the exact same issue and suggested making sure your `react` and `react-dom` are the same version. I tried that and had no luck. Then I stumbled upon another solution for a completely unrelated project but same error, where people suggested deleted your `node_modules` directory and starting over. It worked.

tl;dr: if you're getting this problem, it's probably because something is screwed up in your `node_modules`, which is causing a dependency of `react-datepicker` to fail. Delete the folder and install everything fresh.

**Comment by matYang** [comment]
ran into the same problem, I was updating datepicker from a legacy version of 1.2.7 to 2.7.0

after deleting the datepicker folder under node_modules and running npm install again, it fixed the issue

**Comment by Vidhi-95** [comment]
I deleted the node modules and did npm install again. Still facing the same issue
